### PR TITLE
Fix divide by zero issue in calculating spread priority for zones

### PR DESCRIPTION
**Pull request by hongshibao**
```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go [hunk 1]
```diff
@@ -171,7 +171,10 @@ func (s *SelectorSpread) CalculateSpreadPriority(pod *v1.Pod, nodeNameToInfo map
 		if haveZones {
 			zoneId := utilnode.GetZoneKey(node)
 			if zoneId != "" {
-				zoneScore := float64(schedulerapi.MaxPriority) * ((maxCountByZone - countsByZone[zoneId]) / maxCountByZone)
```
Comment (bsalamat) [inline plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go]: I guess an alternative approach would have been to initialize `maxCountByZone := float64(1)` and leave the this line as it was before, but this is fine too.

**Review by bsalamat — COMMENTED** [review]
Thanks, @hongshibao!

/lgtm

**Comment by bsalamat** [comment]
@hongshibao Please set the release-note to NONE and put the current text in the description of the PR.

/ok-to-test

File: plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go (hunk 1 above)
Comment (hongshibao) [reply plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go]: Nice approach! But I prefer the current one that I feel is more readable. Thanks for review.

**Comment by resouer** [comment]
@bsalamat The approve bot is crazy here... you may need to approve this manually. :)

**Pull request merged**